bears/c_languages/CPPLintBear.py

```diff
@@ -2,6 +2,10 @@
 from coalib.bears.requirements.PipRequirement import PipRequirement
 from coalib.settings.Setting import typed_list
 
+def check_invalid_config(use_spaces, indent_size):
+    if not use_spaces or indent_size != 2:
+        return False
```

Comment:
@Asnelchristian  We need to verify that the user has not changed it from `indent_size=2`.  If they did, the bear will not honour their configuration , and the user should be informed.